### PR TITLE
Support for Oracle Linux 8.

### PR DESCRIPTION
**Pull request by rvva**
Tested on Oracle Linux 8.3 x86_64.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ![Lint](https://github.com/angristan/openvpn-install/workflows/Lint/badge.svg)
 ![visitors](https://visitor-badge.glitch.me/badge?page_id=angristan.openvpn-install)
 
-OpenVPN installer for Debian, Ubuntu, Fedora, CentOS and Arch Linux.
+OpenVPN installer for Debian, Ubuntu, Fedora, CentOS, Oracle Linux 8 and Arch Linux.
```
Comment (angristan) [inline README.md]:
```suggestion
OpenVPN installer for Debian, Ubuntu, Fedora, CentOS, Oracle Linux and Arch Linux.
```

File: openvpn-install.sh [hunk 2]
```diff
@@ -65,6 +65,15 @@ function checkOS() {
 				exit 1
 			fi
 		fi
+		if [[ $ID == "oracle" ]]; then
```
Comment (angristan) [inline openvpn-install.sh]:
```suggestion
		if [[ $ID == "os" ]]; then
```

Maybe?

```
# cat /etc/os-release
NAME="Oracle Linux Server"
VERSION="8.3"
ID="ol"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="8.3"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Oracle Linux Server 8.3"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:oracle:linux:8:3:server"
HOME_URL="https://linux.oracle.com/"
BUG_REPORT_URL="https://bugzilla.oracle.com/"

ORACLE_BUGZILLA_PRODUCT="Oracle Linux 8"
ORACLE_BUGZILLA_PRODUCT_VERSION=8.3
ORACLE_SUPPORT_PRODUCT="Oracle Linux"
ORACLE_SUPPORT_PRODUCT_VERSION=8.3
```

**Review by angristan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rvva) [reply README.md]: fixed in commit [05cf140ffba4c7f5f3811e5563f37bfd13f72373](https://github.com/angristan/openvpn-install/pull/810/commits/05cf140ffba4c7f5f3811e5563f37bfd13f72373)

File: openvpn-install.sh (hunk 2 above)
Comment (rvva) [reply openvpn-install.sh]: fixed in commit [05cf140ffba4c7f5f3811e5563f37bfd13f72373](https://github.com/angristan/openvpn-install/pull/810/commits/05cf140ffba4c7f5f3811e5563f37bfd13f72373)

**Review by angristan — APPROVED** [review]
Thanks!

**Pull request merged**